stories/global.scss

```diff
@@ -66,3 +66,9 @@ table {
   position: inherit!important;
 }
 
+// StringList.stories.mdx
+.string-list-footer {
```

Comment:
Please do not add styling for the components or we are going to have an huge list here. Let's solve the button styling issue by prefixing with `.sbdocs` or `.sidebar-container`